src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/analyticalResults.py

```diff
@@ -0,0 +1,55 @@
+import sys
+sys.path.append('../')
+import numpy as np
+import h5py
+import wellboreAnalyticalSolutions
+
+def analyticalResults(t):
+	ri = 0.1 # inner radius of the wellbore
+	Ti = 100. # temperature applied on the inner surface of the wellbore
+	
+	# Notations and data by Cheng (2016)
+	# Data of Rock salt
+
+	K = 20.7e9 # drained bulk modulus
+	G = 12.4e9 # drained shear modulus
+	beta_d = 3*4e-5 # drained volumetric thermal expansion coefficient of the porous medium
+	Ks = 23.5e9	# bulk modulus of the solid skeleton
```

Comment:
Same here, try to not hard-code these values

Reply:
done